tools/ci_build/github/linux/docker/scripts/install_ubuntu.sh

```diff
@@ -68,27 +65,26 @@ if [ $SYS_LONG_BIT = "64" ]; then
   rm -rf /tmp/dotnet
 fi
 
-if [ $PYTHON_VER != "3.5" ]; then
-    apt-get install -y --no-install-recommends \
-            python${PYTHON_VER} \
-            python${PYTHON_VER}-dev
-    update-alternatives --install /usr/bin/python3 python3 /usr/bin/python${PYTHON_VER} 1
-    update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.5 2
-    update-alternatives --set python3 /usr/bin/python${PYTHON_VER}
-    #TODO: the old one(/usr/bin/pip3) should be uninstalled first. Because the one will be
-    #put at /usr/local/. Then there will be two pips.
-    /usr/bin/python${PYTHON_VER} -m pip install --upgrade --force-reinstall pip==19.0.3
-fi
+#if [ $PYTHON_VER != "3.5" ]; then
+#    apt-get install -y --no-install-recommends \
+#            python${PYTHON_VER} \
+#            python${PYTHON_VER}-dev
+#    update-alternatives --install /usr/bin/python3 python3 /usr/bin/python${PYTHON_VER} 1
+#    update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.5 2
+#    update-alternatives --set python3 /usr/bin/python${PYTHON_VER}
+#    #TODO: the old one(/usr/bin/pip3) should be uninstalled first. Because the one will be
+#    #put at /usr/local/. Then there will be two pips.
+#    /usr/bin/python${PYTHON_VER} -m pip install --upgrade --force-reinstall pip==19.0.3
+#fi
 
-/usr/bin/python${PYTHON_VER} -m pip install --upgrade --force-reinstall numpy==1.15.0
-/usr/bin/python${PYTHON_VER} -m pip install --upgrade --force-reinstall requests==2.21.0
-if [ $DEVICE_TYPE = "Normal" ]; then
-    /usr/bin/python${PYTHON_VER} -m pip install --upgrade --force-reinstall sympy==1.1.1
-fi
-/usr/bin/python${PYTHON_VER} -m pip install --upgrade scipy
+#/usr/bin/python${PYTHON_VER} -m pip install --upgrade --force-reinstall numpy==1.15.0
+/usr/bin/python3 -m pip install --upgrade --force-reinstall numpy==1.15.0
```

Comment:
we need to fix these hacks. otherwise it can't support different versions of PYTHON.